File: Project9/Project9_2/app/src/main/java/com/example/digitclassifier/MainActivity.java
package com.example.digitclassifier;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import com.example.digitclassifier.R;
import com.example.digitclassifier.draw.DrawActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button drwaBtn = findViewById(R.id.drawBtn);
        drwaBtn.setOnClickListener(view -> {
            Intent i = new Intent(MainActivity.this, DrawActivity.class);
            startActivity(i);
        });
    }
}

File: Project3/Project3_2/app/src/main/java/com/example/myapplication4_2/MainActivity.kt
package com.example.myapplication4_2

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.LinearLayout

class MainActivity : AppCompatActivity() {
    private lateinit var btn1: Button
    private lateinit var btn2: Button
    private lateinit var btn3: Button


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        title = "예제 2번"

        btn1 = findViewById<Button>(R.id.Btn1)
        btn2 = findViewById<Button>(R.id.Btn2)
        btn3 = findViewById<Button>(R.id.Btn3)

        btn1.visibility = View.VISIBLE
        btn2.visibility = View.INVISIBLE
        btn3.visibility = View.INVISIBLE


        btn1.setOnTouchListener{ view, motionEvent ->
            btn1.visibility = View.INVISIBLE
            btn2.visibility = View.VISIBLE
            false
        }
        btn2.setOnTouchListener{ view, motionEvent ->
            btn2.visibility = View.INVISIBLE
            btn3.visibility = View.VISIBLE
            false
        }
        btn3.setOnTouchListener{ view, motionEvent ->
            btn3.visibility = View.INVISIBLE
            btn1.visibility = View.VISIBLE
            false
        }
    }
}